src/en/howto-applicationupdateseries.md

```diff
@@ -0,0 +1,113 @@
+Title: Change an application' series post deployment
+
+#  Change an application's series post deployment
+
+## Overview
+
+If using juju 2.3 or later, you can run the `juju update-series` command.
+
+The `juju update-series <application> <series>` will let juju know that future
+units of an application should use the specified series.  
+
+This guide will cover how to do this is previous juju 2.x release.
+
+## Prerequisites
+
+1. Check that the current charm revision in use for the application supports the 
+series you wish to use in the future.  This checked on in the juju charm store.
+
+2. Name of the application.
+
+3. Name of the model where the application lives.
+
+4. Admin privilages to the controller where the model lives.
+
+## 
+
+Start by switching to the juju controller where the changes will happen.
+
+```
+$ juju switch localhost-test:admin/default
+```
+
+Next the database on the controller will need to be modified.  An example script
+can be found [here][appendix].
+
+```
+$ mongo-update-app-series.sh -m default -a ghost -s xenial
```

Comment:
can we package this up as a juju plugin? It could even be a plugin that follow the 2.3 semantics such that if you're on 2.3 it works per docs and if earlier it behaves the same way once the plugin is there?

Reply:
It's not in the cards right now, however it's open to discussion in the future.